### PR TITLE
Add crown on top of the player to indicate number of victories

### PR DESCRIPTION
**Pull request by imeha**
image
image

### CODE REVIEW

File: src/app/player/components/player.component.html [hunk 1]
```diff
@@ -1,3 +1,8 @@
+<div class="player-container" *ngIf="numberOfVictories$ | async as numberOfVictories">>
+  <ng-container *ngIf="numberOfVictories > 0">
```
Comment (DanielMuckelbauer) [inline src/app/player/components/player.component.html:2]: just `numberOfVictories` should suffice I think

**Review by DanielMuckelbauer — APPROVED** [review]

File: src/app/player/components/player.component.html (hunk 1 above)
Comment (imeha) [reply src/app/player/components/player.component.html:2]: I know, I put it in purpose for readability, do you want it badly to be changed?

Comment (DanielMuckelbauer) [reply src/app/player/components/player.component.html:2]: Don't we have it like this in other parts of the code? In my opinion it is more readable and just more elegant, and shorter makes it always easier to understand. I will not insist, I already approved, but I will try to shame you into doing it.

Comment (imeha) [reply src/app/player/components/player.component.html]: Here you go :)

**Pull request merged**